### PR TITLE
fix upload and download job commands, add clone_job

### PR DESCRIPTION
**Pull request by nvkevlu**

### CODE REVIEW

File: nvflare/fuel/hci/server/file_transfer.py [hunk 1]
```diff
@@ -31,6 +32,7 @@
 from nvflare.fuel.hci.conn import Connection
 from nvflare.fuel.hci.reg import CommandModule, CommandModuleSpec, CommandSpec
 from nvflare.fuel.hci.zip_utils import unzip_all_from_bytes, zip_directory_to_bytes
+from nvflare.private.fed.server.server_engine import ServerEngine
```
Comment (YuanTingHsieh) [inline nvflare/fuel/hci/server/file_transfer.py:35]: I think a check to make sure engine is ServerEngine is important as that is the way we ensure engine.job_def_manager exists.

But I think the "fuel" package should not import class from "private"?

Comment (nvkevlu) [reply nvflare/fuel/hci/server/file_transfer.py]: removed the imports for now

Comment (YuanTingHsieh) [reply nvflare/fuel/hci/server/file_transfer.py]: If the file_transfer module depends on server_engine's job_def_manager instance,
should we move the file_transfer module into "private"?

**Review by YuanTingHsieh — APPROVED** [review]

**Comment by YuanTingHsieh** [comment]
It is worth discussing that for upload_job and downlaod_job should we put them here or in the job_cmd.py inside the private package.

**Pull request merged**